src/appshell/qml/DevTools/KeyNav/KeyNavExample.qml

```diff
@@ -45,6 +45,7 @@ Rectangle {
             anchors.margins: 8
             verticalAlignment: Text.AlignVCenter
             text: "Last clicked: " + root.lastClickedInfo
+            isKeyNav: true
```

Comment:
In this case, it would be better to write just `color: #000000` in this file. 

Since `color` is a property of `StyledTextLabel` (because that is a subclass of `Text`), you can set it directly in places where StyledTextLabel is instantiated. 
Because of this, and because of the fact that you need to change this property in only one specific case, it is not worth a dedicated `isKeyNav` property. So you don't need to make any changes to StyledTextLabel.qml.

Reply:
Done